### PR TITLE
Publish packages from CI

### LINKED ISSUE #950: Publish packages from CI
See https://openastronomy-azure-pipelines.readthedocs.io/en/latest/publish.html#usage-and-options.

### PR DESCRIPTION
**Pull request by epassaro**
- [ ] Use the OpenAstronomy `publish` template
- [ ] Set PyPI endpoint 
- [ ] Use `test_extras` and `test_commads` in template to test wheels

Closes #950

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -29,57 +29,74 @@ trigger:
...
+            - macos: py38
+              name: py38_test_macos
+              libraries: {}
+
+  - stage: publish
+    jobs:
+      - template: publish.yml@OpenAstronomy
+        parameters:
+          ${{ if startsWith(variables['Build.SourceBranch'], 'refs/tags/v') }}:
+            pypi_connection_name: 'pypi_endpoint'
```
Comment (astrojuanlu) [inline azure-pipelines.yml:94]:
```suggestion
            pypi_connection_name: 'pypi_upload'
```

File: azure-pipelines.yml [hunk 2]
```diff
@@ -29,57 +29,74 @@ trigger:
...
+          libraries:
+            apt:
+              - pandoc texlive texlive-latex-extra texlive-fonts-recommended dvipng graphviz cm-super-minimal
```
Comment (astrojuanlu) [inline azure-pipelines.yml:97]: I don't think we need the LaTeX dependencies to create the sdists and wheels, but I might be wrong?

**Review by astrojuanlu — CHANGES_REQUESTED** [review]
Apart from a couple of minor comments, this is almost good to go!

I noticed that these commits don't have an associated email though, and therefore they are not linked to @epassaro. Would you like to

- Squash all these commits into one, and
- Configure your email so the commit is properly attributed?

File: azure-pipelines.yml (hunk 2 above)
Comment (epassaro) [reply azure-pipelines.yml]: Seems right!

**Comment by epassaro** [comment]
I messed up this PR trying to squash, I'll make a new one, sorry @astrojuanlu.

**Pull request closed without merging**